<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aiueoSubWin</class>
 <widget class="QWidget" name="aiueoSubWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>82</width>
    <height>74</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>82</width>
    <height>74</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>82</width>
    <height>74</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
   <item>
    <widget class="QLabel" name="romanYin">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pingjiaName">
       <property name="font">
        <font> 
         <family>等线</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>あ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pianjiaName">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ア</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
